pandas/errors/__init__.py

```diff
@@ -455,12 +455,14 @@ class CSSWarning(UserWarning):
     Examples
     --------
     >>> df = pd.DataFrame({'A': [1, 1, 1]})
-    >>> (df.style.applymap(lambda x: 'background-color: blueGreenRed;')
-    ...         .to_excel('styled.xlsx')) # doctest: +SKIP
+    >>> df.style.applymap(
+    ...     lambda x: 'background-color: blueGreenRed;'
+    ... ).to_excel('styled.xlsx')  # doctest: +SKIP
     ... # CSSWarning: Unhandled color format: 'blueGreenRed'
-    >>> (df.style.applymap(lambda x: 'border: 1px solid red red;')
-    ...         .to_excel('styled.xlsx')) # doctest: +SKIP
-    ... # CSSWarning: Too many tokens provided to "border" (expected 1-3)
+    >>> df.style.applymap(
+    ...     lambda x: 'border: 1px solid red red;'
+    ... ).to_excel('styled.xlsx')  # doctest: +SKIP
+        CSSWarning: Unhandled color format: 'blueGreenRed'
```

Comment:
Did you intend to remove ... # in this line?

Otherwise lgtm

Reply:
I just moved them back as technically they're part of the output, rather than the doctest